wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/TextDocumentService.java

```diff
@@ -440,12 +440,28 @@ public boolean canDo(ExtendedLanguageServer element) {
           public boolean handleResult(ExtendedLanguageServer element, Hover hover) {
             if (hover != null) {
               HoverDto hoverDto = new HoverDto(hover);
-              result.getContents().addAll(hoverDto.getContents());
+              Either<List<Either<String, MarkedString>>, MarkupContent> contents =
+                  hoverDto.getContents();
+              if (contents.isLeft()) {
+
```

Comment:
do we need to make some rendering of the content here?

Reply:
Yes we do...fixing the test does not make the code correct.